apps/wallet/src/engine/runners/solana/history/index.ts

```diff
@@ -0,0 +1,428 @@
+import type {
+	Connection,
+	ParsedInstruction,
+	ParsedTransaction,
+	ParsedTransactionMeta,
+	ParsedTransactionWithMeta,
+	TokenBalance,
+} from '@solana/web3.js';
+import type { PublicKey } from '@solana/web3.js';
+import type {
+	SolanaSwapHistory,
+	SolanaSwapHistoryV2,
+	SolanaToken,
+	SolanaTransferHistory,
+	SolanaTransferHistoryV2,
+	SolanaUnknownHistory,
```

Comment:
V2 will be the current used types, so you could deprecate the old types with `v1` prefix and remove `v2` of current types